script.js

```diff
@@ -0,0 +1,65 @@
+// 1 задачв
+let firstName = prompt('Як Вас звати?');
+alert('Привіт,' + firstName);
+
+// 2 задача
+let yourage = prompt('В якому році Ви народились?');
+const year = 2020;
+
+let result = year - yourage; 
+
+alert(result);
+
+// 3 задача
+let lenght =prompt ('Вкажіть довжину сторони квадрата');
+alert(lenght * lenght);
+
+// 4 задача
+let radius =prompt ('Вкажіть радіус кола');
+const pi =3.14;

```

Comment:
[https://www.google.com/search?sxsrf=ALeKk00PUdJPFkUI25CRCKawQdJE1aZb0w%3A1592231700035&ei=FIfnXsncAfvWmwXzwpjwBw&q=%D1%87%D0%B8%D1%81%D0%BB%D0%BE+%D0%BF%D0%B8+js&oq=%D1%87%D0%B8%D1%81%D0%BB%D0%BE+%D0%BF%D0%B8+&gs_lcp=CgZwc3ktYWIQARgJMgIIADICCAAyAggAMgIIADICCAAyAggAMgIIADICCAAyAggAMgcIABAUEIcCOgQIABBHUMxLWMxLYOReaABwAXgAgAFhiAFhkgEBMZgBAKABAaoBB2d3cy13aXo&sclient=psy-ab](url)